SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -48,9 +48,26 @@ void USpatialGDKSettings::PostInitProperties()
 {
 	Super::PostInitProperties();
 
-	// Check any command line overrides for using QBI (after reading the config value):
+	// Check any command line overrides for using QBI, Offloading (after reading the config value):
 	const TCHAR* CommandLine = FCommandLine::Get();
-	FParse::Bool(CommandLine, TEXT("useQBI"), bUsingQBI);
+
+	if (FParse::Param(CommandLine, TEXT("OverrideSpatialOffloading")))
+	{
+		bEnableOffloading = true;
+	}
+	else
+	{
+		FParse::Bool(CommandLine, TEXT("OverrideSpatialOffloading="), OUT bEnableOffloading);
```

Comment:
Although I'm personally in favor of using `OUT` to signify out parameters, we should probably first all agree on using this, and then do a sweep of adding these to relevant places as opposed to just adding them in new PRs. Otherwise we'll just have arbitrary parts of the code using or not using these.

Reply:
Good point. Removed.